components/leaderboard/BadgesCollection.tsx

```diff
@@ -1,15 +1,68 @@
-import React from 'react';
+import React, { Fragment, useState } from 'react';
 import Image from 'next/image';
+import { Dialog, Transition } from '@headlessui/react';
+import { CheckIcon } from '@heroicons/react/24/outline';
 
 export default function BadgesCollection(props) {
+  const [showModal, setShowModal] = React.useState(false);
```

Comment:
`React.` is redundant here since u imported `useState` alr

Reply:
Done!